packages/server/src/users.ts

```diff
@@ -38,10 +50,10 @@ export async function getUserByName(username: string): Promise<PersistentUser> {
 }
 
 export async function getUserBySessionId(
-  id: string,
+  session_id: string,
```

Comment:
Nice! (I had assumed it might be clear from the function name, but never hurts to be more explicit)

Reply:
Yep! I will do the same for other functions that take id as a parameter